src/argilla/client/feedback/schemas/records.py

```diff
@@ -280,6 +399,40 @@ def set_suggestions(
         )
         self.update(suggestions=suggestions)
 
+    @allowed_for_roles(roles=[UserRole.owner, UserRole.admin])
+    def delete_suggestions(self, suggestions: Union[RemoteSuggestionSchema, List[RemoteSuggestionSchema]]) -> None:
+        """Deletes the provided suggestions from the record in Argilla. Note that the
+        suggestions must exist in Argilla to be removed from the record.
+
+        Args:
+            suggestions: can be a single `RemoteSuggestionSchema` or a list of
+                `RemoteSuggestionSchema`.
+
+        Raises:
+            PermissionError: if the user does not have either `owner` or `admin` role.
+        """
+        if isinstance(suggestions, RemoteSuggestionSchema):
+            suggestions = [suggestions]
+
+        existing_suggestions = {suggestion.question_name: suggestion for suggestion in self.suggestions}
+        delete_suggestions = []
+        for suggestion in suggestions:
+            if suggestion.question_name not in existing_suggestions:
+                warnings.warn(
+                    f"A suggestion for question `{suggestion.question_name}` has not been"
+                    " provided, so it cannot be removed.",
+                    UserWarning,
+                    stacklevel=1,
+                )
+            else:
+                existing_suggestions.pop(suggestion.question_name, None)
+                delete_suggestions.append(suggestion)
+
+        records_api_v1.delete_suggestions(
+            client=self._client, id=self.id, suggestion_ids=[suggestion.id for suggestion in delete_suggestions]
+        )
```

Comment:
maybe handle possible exceptions here and reraise with `RuntimeError` like we do in other parts of the codebase